packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -87,6 +89,7 @@ const DOM = props => {
 				{newrelicJS &&
 					<script dangerouslySetInnerHTML={getInnerHTML(newrelicJS)} />}
 				<script dangerouslySetInnerHTML={getInnerHTML(uxCaptureJS)} />
+				<script dangerouslySetInnerHTML={getInnerHTML(fontLoaderJS)} />
```

Comment:
looks like this will render

```html
<script>
  <script src="..." />
</script>
```

I think `fontLoaderJS` should probably be `fontLoaderSrc`, and then this line would be

```html
<script src={fontLoaderSrc} />
```

Reply:
For this one, I hope to actually replace it with inline code instead, once it all works out because timing for network request can skew this way too much.

I might need to look into how to minimize the size of the library first (although it's only 6K on the wire).